src/dotnet/commands/dotnet-new/CSharp_Console1.1/Program.cs

```diff
@@ -1,9 +1,12 @@
 using System;
 
-class Program
+namespace ConsoleApp
```

Comment:
Per the other templates (like [class library](https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-new/CSharp_Lib/Library.cs#L3) for instance), this is hard coded. The corresponding `new3` templates use the appropriate variants on the user supplied name for file/namespace/item names.